internal/pkg/cli/validate.go

```diff
@@ -96,3 +99,50 @@ func validateGitHubRepo(val interface{}) error {
 	}
 	return nil
 }
+
+// validateDockerfiles returns the list of Dockerfiles within the current
+// working directory and a sub-directory level below. If an error occurs while
+// reading directories, or no Dockerfiles found returns the error.
+func validateDockerfiles(fs afero.Fs) ([]string, error) {
```

Comment:
I prefer the old function name as it's more indicative of what the function actually does. `listDockerfiles`, `listDockerfileDirs`, or something similar.

Reply:
I agree "listDockerfiles" may be better. However, if I change the func name, then this func seems not suitable to be placed in "validation.go". I am wondering if we have another place for a package-level available function.

Reply:
Maybe `dockerfiles.go` in the `cli` package?

Reply:
Sounds like a good idea :+1: